malariagen_data/anopheles.py

```diff
@@ -6655,20 +6655,24 @@ def plot_haplotype_network(
         server_mode: Optional[
             dash_params.server_mode
         ] = dash_params.server_mode_default,
-        height: dash_params.height = 650,
+        height: dash_params.height = 600,
         width: Optional[dash_params.width] = "100%",
+        serve_scripts_locally: dash_params.serve_scripts_locally = dash_params.serve_scripts_locally_default,
     ):
         import dash_cytoscape as cyto
-        from dash import dcc, html
+        from dash import Dash, dcc, html
         from dash.dependencies import Input, Output
-        from jupyter_dash import JupyterDash
+
+        debug = self._log.debug
+
+        # https://dash.plotly.com/dash-in-jupyter#troubleshooting
+        # debug("infer jupyter proxy config")
+        # Turn off for now, this seems to crash the kernel!
+        # from dash import jupyter_dash
+        # jupyter_dash.infer_jupyter_proxy_config()
```

Comment:
This doesn't seem to work at all, so leaving commented out. It's not necessary when working on colab or on a local jupyterlab server.